templates/base.html

```diff
@@ -262,12 +262,24 @@
           >Categories</label>
           <select
             id="categories-list"
-            class="select select-bordered select-sm max-w-xs dm-sans"
+            class="select select-bordered select-sm dropdown max-w-xs dm-sans"
           >
             <option value="">All</option>
             {% for category in categories %}<option value="{{ category }}">{{ category }}</option>{% endfor %}
           </select>
         </div>
+        <div class="text-start ml-[2em] xl:ml-[4em]">
+          <label
+            for="search-bar"
+            class="hidden lg:inline-block kiwix-{{page }}-filter-label text-[0.9em] me-[1em] dm-sans"
+          >Search</label>
```

Comment:
`Find` as label seems more appropriate (albeit not perfect). Search word sort of implies it will look at content while this is just a free-text filter on title/desc.
@siemsie any suggestion on the label ?